frontend/src/components/buttons/Button.css

```diff
@@ -0,0 +1,20 @@
+.global-button {
```

Comment:
To keep a maintainable codebase and following best practices - use styled components instead unless it is absolutely necessary to use an external CSS file.